community/examples/intel/README.md

```diff
@@ -1,454 +0,0 @@
-# Intel Solutions for the Cluster Toolkit (formerly HPC Toolkit)
```

Comment:
Please leave a "trail of crumbs" for future customers with the recommendations you mention in the PR description:

- migrate from DAOS to 1st-party Parallelstore
  - link to module
  - link to example
- if 3rd-party DAOS module is required, link to last Toolkit release (v1.41.0?)